user/deployment/snaps.md

```diff
@@ -1,38 +1,52 @@
-      on:
-        branch: master
-    ```
+In order for Travis CI to have permission to upload the snap on your behalf, you need to export a Snap Store login token. You can do this with the snapraft command-line tool, once you have [enabled snap support](https://docs.snapcraft.io/core/install) on your system.
 
-The `snapcraft enable-ci travis` command will assist you getting the credential, encrypting it and adding the right script to the deploy section of your `.travis.yml` file. Here is a [tutorial that will guide you setting up the continuous delivery from Travis CI](https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#0) on your project.
+```bash
+sudo snap install snapcraft --classic
+```
 
-Note that the `edge` channel is intended for crowdtesting with your community of early adopters. With this deployment script in Travis, every time a pull request lands into the master branch, a new snap will be published to edge. Your testers can install it in any of the supported Linux distros with:
+Login tokens can attenuate how, when, and where they can be used, thus minimising damage from compromise. For Travis CI, export a token that can only upload this snap to the channel you specified above (in this example, `edge`):
 
 ```bash
-sudo snap install my-snap-name --edge
+snap export-login --snaps my-snap-name --channels edge -
 ```
 
-After that, they will be always ready to provide early feedback and help making a more stable release.
+_Note: The final `-` requests the login be exported to stdout instead of a file. It is required._
+
+The token will be printed out. Copy and put it into the Travis CI environment variable `$SNAP_TOKEN`:
+
+```bash
+travis env set SNAP_TOKEN "<token>"
+```
 
-You can adjust the Travis script and the `snapcraft` calls to fit your development process. For example, you could install the snap and run user acceptance tests before releasing to edge, to make sure that there are no regressions in master. Also, there are three other channels in the store: `beta`, `candidate` and `stable`. You can run different tests suites with Travis for each channel on your way to stable.
+Note: The `edge` channel is intended for the bleeding edge: your every commit to master will be built and uploaded. This is great for crowd-testing with your community of early adopters, who can install your app in any of the [supported Linux distributions](https://docs.snapcraft.io/core/install) with:
 
-There are plenty of [tutorials](https://tutorials.ubuntu.com/) and [videos](https://www.youtube.com/snapcraftio) that will help you packaging your project as a snap.
+```bash
+sudo snap install my-snap-name --edge
+```
+
+Each upload gets a monotonically increasing integer. When you're ready, you can release one of these built commits to the stable channel for the public to discover in the [Snap storefront](https://snapcraft.io/store). For example, you could promote the very first upload to stable:
```

Comment:
Sorry, I know I said "the public to discover", but what do you think of "for public discovery in the"?

Reply:
Sure, done.